src/provider/provider_tracking.c

```diff
@@ -76,6 +127,96 @@ static umf_result_t umfMemoryTrackerAdd(umf_memory_tracker_handle_t hTracker,
     return UMF_RESULT_ERROR_UNKNOWN;
 }
 
+static tracker_alloc_info_t *
+critnib_get_last(umf_memory_tracker_handle_t hTracker, const void *ptr,
```

Comment:
I think the name of the function should not include `critnib` because it is just underlying data structure used by memory tracker.

Reply:
Done